charts/rekor/templates/backfillredis/backfill-job.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.backfillredis.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "rekor.backfillredis.fullname" . }}
+{{ include "rekor.namespace" . | indent 2 }}
+  labels:
+    {{- include "rekor.backfillredis.labels" . | nindent 4 }}
+{{- if .Values.backfillredis.annotations }}
+  annotations:
+{{ toYaml .Values.backfillredis.annotations | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.backfillredis.ttlSecondsAfterFinished }}
+  ttlSecondsAfterFinished: {{ .Values.backfillredis.ttlSecondsAfterFinished }}
+{{- end }}
+  template:
+    spec:
+      serviceAccountName: {{ template "rekor.serviceAccountName.backfillredis" . }}
+      restartPolicy: Never
+      automountServiceAccountToken: {{ .Values.backfillredis.serviceAccount.mountToken }}
+      containers:
+        - name: {{ template "rekor.backfillredis.fullname" . }}
+          image: "{{ template "rekor.image" .Values.backfillredis.image }}"
+          imagePullPolicy: "{{ .Values.backfillredis.image.pullPolicy }}"
+          args: [
+            "--address={{ template "redis.hostname" . }}",
+            "--port={{ .Values.backfillredis.redisPort }}",
```

Comment:
Can we get the port from the redis config? Or is there something in the redis config that combines hostname and port? I can update the script's flags